packages/account-v2/src/pages/ConnectedApps/ConnectedAppsResponsive.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { OauthApps } from '@deriv/api-types';
+import { Button } from '@deriv-com/ui';
+import { formatDate, getFormattedAppScopes } from '../../utils/connectedAppsUtils';
+import { ConnectedAppsResponsiveColumn } from './ConnectedAppsResponsiveColumn';
+
+type TConnectedAppsResponsive = {
+    connectedApps: OauthApps;
+    handleToggleModal: (appId: number) => void;
+};
+
+export const ConnectedAppsResponsive = ({ connectedApps, handleToggleModal }: TConnectedAppsResponsive) => (
+    <div className='flex flex-col gap-8'>
+        {connectedApps.map(connectedApp => (
+            <div className='bg-solid-grey-2 rounded-default p-16' key={connectedApp.app_id}>
+                <div className='flex justify-between gap-16'>
+                    <ConnectedAppsResponsiveColumn description={connectedApp?.name} title='Name' />
+                    <ConnectedAppsResponsiveColumn
+                        description={formatDate(connectedApp?.last_used)}
+                        style='w-80'
+                        title='Last Login'
+                    />
+                </div>
+                <div className='flex justify-between gap-16'>
+                    <ConnectedAppsResponsiveColumn
+                        description={getFormattedAppScopes(connectedApp.scopes)}
```

Comment:
should we use connectedApp?.scopes

Reply:
Not sure, maybe we can remove `.?` since, as @arshad-rao-deriv pointed it might never be undefined. What do you say?